installing/installing_oci/installing-oci-assisted-installer.adoc

```diff
@@ -0,0 +1,55 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="installing-oci-assisted-installer"]
+= Using the Assisted Installer to install a cluster on OCI
+include::_attributes/common-attributes.adoc[]
+:context: installing-oci-assisted-installer
+
+toc::[]
+
+From {product-title} {product-version} and later versions, you can use the Assisted Installer to install a cluster on {oci-first} by using infrastructure that you provide.
+
+// The Assisted Installer and OCI overview
+include::modules/installing-oci-about-assisted-installer.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+* link:https://access.redhat.com/documentation/en-us/assisted_installer_for_openshift_container_platform/2024[Assisted Installer for {product-title}]
+* xref:../../installing/installing_platform_agnostic/installing-platform-agnostic.html#cluster-entitlements_installing-platform-agnostic[Internet access for {product-title}]
+* link:https://docs.oracle.com/en-us/iaas/Content/Block/Concepts/blockvolumeperformance.htm#vpus[Volume Performance Units]
+ 
+
+// Creating OCI resources
+include::modules/creating-oci-resources-services.adoc[leveloffset=+1]
+
+[role="_additional-resources"]
+.Additional resources
+
+See the following Oracle web-based documents:
+
+* link:https://docs.oracle.com/en-us/iaas/Content/Identity/Tasks/managingcompartments.htm[Managing compartments]
+* link:https://docs.oracle.com/en-us/iaas/Content/Object/Concepts/objectstorageoverview.htm[Overview of Object Storage]
+
+// Using the Assisted Installer to generate an OCI-compatible Agent ISO image
+include::modules/using-assisted-installer-oci-agent-iso.adoc[leveloffset=+1]
+
+// Downloading manifest files and deployment resources
+include::modules/downloading-manifest-files-resources-oci.adoc[leveloffset=+1]
+
+// Completing the remaining Assisted Installer steps
+include::modules/complete-assisted-installer-oci.adoc[leveloffset=+1]
+
+// Verifying a succesful cluster installation on OCI
```

Comment:
```suggestion
// Verifying a successful cluster installation on OCI
```

Reply:
VALE did not catch this.....ARRRRG!

Reply:
LOL. Spelling this word hurts my eyes! Fixed.